exodus/reports/templates/reports_list.html

```diff
@@ -8,9 +8,14 @@
     {% if reports %}
         <div style="text-align:center">
             <h1 class="main-title">
-                {{ count }} {% trans "Report" %}{{ count | pluralize }}
-                {% if handle %}<small>({{ handle }})</small>{% endif %}
+                {% trans "Reports" %}
             </h1>
+            <h5>
+                {{ reports_count }} {% trans "report" %}{{ reports_count | pluralize }}
+                {% trans "for" %}
+                {{ apps_count }} {% trans "application" %}{{ apps_count | pluralize }}
+                {% if handle %}<small>({{ handle }})</small>{% endif %}
+            </h5>
```

Comment:
The second part is not good in the case of a single application.
ex: https://reports.exodus-privacy.eu.org/en/reports/search/byook.ltlm/

I'd do:
```
            <h1 class="main-title">
                {% trans "Reports" %}
            </h1>
            <h4>
              {% if handle %}
                {{ handle }}
              {% else %}
                {{ reports_count }} {% trans "report" %}{{ reports_count | pluralize }}
                {% trans "for" %}
                {{ apps_count }} {% trans "application" %}{{ apps_count | pluralize }}
              {% endif %}
            </h4>
```

Reply:
Wow, had no idea this template called "list" was also used for a search page!
How do you reach this search? The only link to this search I see was... On the App page I'm removing =)
Shouldn't I better delete `search/(?P<handle>.+)/` route and simplify this page to remove the "search" thing?

Reply:
This page is accessible **when you search for "other versions"** of a report (now by clicking on the version link from a given report). This was also the link given by the search engine on the search/home page previously but was changed recently to link directly to the latest report instead.

We definitely need to keep this page (although we could change its URL, but that's another topic :))

Reply:
Alright then, fixed.